scripts/quickstart.sh

```diff
@@ -290,6 +292,16 @@ function parseArguments {
       echo "Warning! trying to run a bootnode without BOOTNODES argument!"
     fi
   fi
+  
+  if [ -z "$TESTNET" ] ; then
+    if [ $TESTNET == "true" ]
+      if [[ $ROLE == bridge-validator ]] ; then
+        displayErrorAndExit "bridge-validators not supported on Spark"
+      fi
+      TEST_NET_STRING = "-spark"
+      VERSION_FILE = "https://raw.githubusercontent.com/fuseio/fuse-network/master/Version_testNet"
```

Comment:
can you rename the version file to `Version_spark-testnet` to be aligned with test of the naming